dotcom-rendering/src/lib/generateImageURL.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Generates a URL for calling the Fastly Image Optimiser.
+ *
+ * @see https://developer.fastly.com/reference/io/
+ * @see https://github.com/guardian/fastly-image-service/blob/main/fastly-io_guim_co_uk/src/main/resources/varnish/main.vcl
+ *
+ */
+export const generateImageURL = ({
```

Comment:
Not sure what the benefit of extracting this to a lib versus exporting from `Picture` directly. I would also suggest only using `generateSources`.